tests/test_connection.py

```diff
@@ -8,6 +11,21 @@
 from psycopg3.conninfo import conninfo_to_dict
 
 
+@pytest.fixture
+def temp_table(svcconn):
```

Comment:
This is too generic for such a specific `id text primary key` result.

Probably this fixture should return an object to create temp tables, so that tests would be:
```python
def test(temp_table):
    temp_table.create("foo", "id int primary key")
    # ...
```

Reply:
Also note that if several tests, e.g. all the tests of the same module, require the same, or few tables definitions, they can create them into a fixture.

```python
@pytest.fixture
def my_table(temp_table):
    temp_table.create("foo", "id int primary key")
    return temp_table
```

Reply:
Does this objection go away if this fixture is moved to the more specific `test_transaction.py` module?

Or would you prefer I turn it into a fixture-factory, and hoist it out into `fix_db.py` so that tests in `test_connection.py` (and potentially elsewhere) can continue to use it?

If so, any preference on what to call the 'local' fixture? `temp_table_with_single_id_column_of_type_int`? :-p
(For `test_transaction.py` I'd consider making it an auto fixture, since most tests require it anyway -- this way the name becomes less of an issue as it doesn't need to be mentioned in every test signature.)

Reply:
Yes: the objection goes away if the tests go into a `test_transaction.py` table :) It only re-proposes itself in case you start needing more than one different table into the same module.

We can introduce the fixture-factory at a later time. I kinda liked `temp_table` for the fixture name but it's also kinda wrong (that is not a table, and is not a temp table either) so I'd probably call the fixture `tables`.

However don't implement it in this MR unless you find yourself wanting to use it.

Reply:
How about this implementation: https://github.com/asqui/psycopg3/commit/f2e85b9914c13a5ee821cc4cfdb2e84b65c693cc
(This is a commit on a separate branch to show a cleaner diff of net changes to the original `test_connection.py`)